src/components/google-maps/container.ts

```diff
@@ -0,0 +1,115 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import GoogleMapsApi from 'load-google-maps-api';
+import { getCurrentGeoLocation } from './helpers';
+
+// Require the marker image to show the current position
+const currentPosMarkerImg = require('./current-position-marker.png');
+
+@Component
+export default class GoogleMapContainer extends Vue {
+  // The Google Maps API Key
+  @Prop({
+    type: String,
+    required: false
+  })
+  readonly apiKey!: string;
+
+  // The Google Maps Configuration
+  @Prop({
+    type: Object,
+    required: false,
+    default: () => ({
+      center: { lat: 0, lng: 0 },
+      zoom: 18
+    })
+  })
+  readonly config!: google.maps.MapOptions;
+
+  // The google map object
+  // google: google.maps | null = null;
+  // TODO: fix TS error that doesn't allow google.maps as a type
+  google!: any;
+
+  // The map object
+  map!: google.maps.Map;
+
+  // The current position
+  currentPos: google.maps.LatLngLiteral = {
+    lat: 0,
```

Comment:
Interesting. Whats it mean in TS when a type is `0`? (`lat: 0`). Should this be `lat: number`?

Reply:
This is actually setting the `lat` value to be `0`. not setting a type. The type is `google.maps.LatLngLiteral`.

Reply:
ooooooooooooh. ha ha ha ha.

Reply:
We're inside a property. I missed that detail when commenting. Duh